.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,243 @@
+
+Examples of commit messages following Conventional Commits:
+
+- `fix(charts/CHART): chart repo url points to the right location`
+- `feat(charts/CHART): add external secret`
+- `docs(charts/CHART): add note about private GKE clusters`
+
+Note the added `charts/CHART` context in the commit message: it signals which chart is affected.
+
+Please look at the [Conventional Commits site](https://www.conventionalcommits.org/) for more examples and details.
+
+
+### Documenting changes
+
+If you made changes to the documentation template (`README.md.gotmpl`) or `values.yaml`,
+you need to regenerate the documentation:
+
+```bash
+make docs
+git commit -m 'docs(charts/CHART): regenerate documentation'
+```
+
+You can also bundle the documentation changes with your other commits to make the commit history nicer.
+
+
+### Updating `Chart.yaml`
+
+After all changes are done, you need to update the `Chart.yaml` file in the chart.
+
+
+#### Adding change log
+
+Changes documented under the `artifacthub.io/changes` annotation appear automatically on [Artifact Hub](https://artifacthub.io/).
+
+The annotation value for each version (and as a result each PR) should be cleared before adding new entries.
+
+Here is an example for a change log:
+
+```yaml
+annotations:
+  artifacthub.io/changes: |
+    - kind: added
+      description: Awesome feature that will let you do X
+      links: # Optional
+        - name: Github PR
+          url: https://github.com/.../.../pull/1234
+```
+
+The kind can be one of the following:
+
+- added
+- changed
+- deprecated
+- removed
+- fixed
+- security
+
+The description should be brief, but explain the change in as much detail as possible.
+If needed, you can add links to the GitHub PR or other related resources to add more detail.
+
+
+#### Bumping chart version
+
+Every merged change automatically gets released with the version found in `Chart.yaml`.
+As a result, every PR needs to bump the version in that file.
+Failing to do that results in CI failure.
+
+After bumping the version regenerate the documentation:
+
+```bash
+# bump chart version
+make docs
+git commit -m 'chore(charts/CHART): bump chart version'
+```
+
+
+### Opening the pull request
+
+Once you are done with all changes, push your branch to GitHub:
+
+```bash
+git push -u origin my-branch
+```
+
+Then click on the pull request link appearing in the terminal or go to your fork and
+open a pull request from your branch.
+
+Fill out the pull request template and hit submit.
+
+Wait for someone to review your changes.
+Please don't start pinging maintainers if noone reacts in a couple days:
+we are all busy with our life.
+
+Once your changes are reviewed the pull request either get merged
+or the reviewer requests some changes. We try to minimize the number of review rounds,
+but please don't get offended if you need to update your pull request multiple times.
+
+Eventually your changes will be merged. Congratulations and thanks! 🎉👍
```

Comment:
My opinion: I'd drop the "Eventually your changes will be merged." line -- I appreciate that you're trying to encourage the audience, but it may not actually be merged in the end.

```suggestion
Congratulations on the contribution and thanks! 🎉👍
```

Reply:
Yeah, I wasn't sure about that part. :)